src/fitnesse/FitNesseExpediter.java

```diff
@@ -69,13 +69,9 @@ public void run() {
   }
 
   @Override
-  public void send(byte[] bytes) {
-    try {
-      output.write(bytes);
-      output.flush();
-    } catch (IOException e) {
-      LOG.log(Level.FINE, format("Could not send data for URL %s: %s (Stop button pressed?)", (request != null ? request.getResource() : ""), e.toString()));
```

Comment:
There is already an issue for this: #834. I expect it has something to do with browsers (most notably Chrome) prefetching data.